.ci/bump-license-year.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+PREV_YEAR=$1
+CURR_YEAR=$2
+DIR=$3
+
+OLD_VALUE="// Copyright (C) 2001-$PREV_YEAR the original author or authors."
+NEW_VALUE="// Copyright (C) 2001-$CURR_YEAR the original author or authors."
+
+find $DIR -type f \( -name *.java -o -name *.header -o -name *.g \) \
+  -exec sed -i "s|$OLD_VALUE|$NEW_VALUE|g" {} +
+
+BASEDIR=$(pwd)
+echo "Distict Diff in $DIR is:"
```

Comment:
Should this be Distinct?

I think it would be better to make some wording like last time, like "Diff should only be 1 line and should not continue if it is more than 1".

Reply:
typo is fixed.
I try to avoid validation ... script should just work. I keep output to have some logs of what happened for investigation if problem appear.